Doc/library/datetime.rst

```diff
@@ -121,7 +121,7 @@ Available Types
 .. class:: timedelta
    :noindex:
 
-   A duration expressing the difference between two :class:`date`, :class:`.time`,
+   A duration expressing the difference between two :class:`date`
```

Comment:
In my opinion, the text in the _datetime Objects_ section is also confusing:

> A `timedelta` object represents a duration, the difference between two dates or times.

since it's ambiguous whether "times" refers to `datetime.datetime`, `datetime.time`, or both.

I'd like to also clarify that in this PR, if someone agrees.

Reply:
Yes, please clarify it as well.

I suggest also to place `datetime` at first place, because the difference between `datetime` objects is the primary purpose of `timedelta`, and this is how it is commonly used.

Reply:
done, thank you